tests/house_tests.py

```diff
@@ -148,42 +142,3 @@ def form_data_filter_tests():
         'lowest_income': 5000,
         'children': 1,
     }
-
-
-@pytest.mark.django_db
-class TestMyHouseViews:
-    def test_house_view_function_200(self, client, new_user):
-        client.force_login(new_user)
-        HouseFactory(user=new_user).save()
-        response = client.get('/house/')
-        assert response.status_code == 200
-
-    def test_mine_page_expenses_table_views_templates(self):
-        try:
-            get_template(MINE_EXPENSES_TITLE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_EXPENSES_TITLE_ROUTE} does not exist"
-
-    def test_mine_page_house_table_views_templates(self):
-        try:
-            get_template(MINE_HOUSE_TABLE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_HOUSE_TABLE_ROUTE} does not exist"
-
-    def test_mine_page_house_table_title_views_templates(self):
-        try:
-            get_template(MINE_HOUSE_TABLE_TITLE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_HOUSE_TABLE_TITLE_ROUTE} does not exist"
-
-    def test_mine_page_mine_page_views_templates(self):
-        try:
-            get_template(MINE_MINE_PAGE_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_MINE_PAGE_ROUTE} does not exist"
-
-    def test_mine_sidebar_views_templates(self):
-        try:
-            get_template(MINE_SIDEBAR_ROUTE)
-        except TemplateDoesNotExist:
-            assert False, f"Template {MINE_SIDEBAR_ROUTE} does not exist"
```

Comment:
Don't we need these tests?

Reply:
my bad, I see that you move it to another file

Reply:
yes
I moved it